docs/getting-started.md

```diff
@@ -12,9 +12,14 @@ registered:
 - `Microsoft.EventGrid`
 - `Microsoft.Network`
 - `Microsoft.Compute`
+- `Microsoft.SignalRService`
 
 ## Deploying an instance of Onefuzz
 
+Ensure you have python3 (at least 3.7), [Azure Functions Core Tools
```

Comment:
It looks like the the deploy scripts (like `deply.py`) have an interpreter of `#!/usr/bin/env python`, which makes this potentially misleading on older systems, given the instructions below. We want `python` to alias to `python3.7` or greater, not any `python2`.

How about something like:
```suggestion
Ensure you have Python with `python --version` >= 3.7, [Azure Functions Core Tools
```